image-gallery.html

```diff
@@ -0,0 +1,196 @@
+  <meta name="twitter:image" content="http://i.imgur.com/BBVehHB.png">
+  <meta name="twitter:url" content="http://www.lnug.org">
+  <meta name="google-site-verification" content="gnZ_Oz6_k7Gv93jmtpCeY6Rjz6E_rjh_1n5dlYgMcJw">
+
+  <link rel="apple-touch-icon-precomposed" sizes="57x57" href="./images/favicon/apple-touch-icon-57x57.png">
+  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="./images/favicon/apple-touch-icon-114x114.png">
+  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="./images/favicon/apple-touch-icon-72x72.png">
+  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="./images/favicon/apple-touch-icon-144x144.png">
+  <link rel="apple-touch-icon-precomposed" sizes="60x60" href="./images/favicon/apple-touch-icon-60x60.png">
+  <link rel="apple-touch-icon-precomposed" sizes="120x120" href="./images/favicon/apple-touch-icon-120x120.png">
+  <link rel="apple-touch-icon-precomposed" sizes="76x76" href="./images/favicon/apple-touch-icon-76x76.png">
+  <link rel="apple-touch-icon-precomposed" sizes="152x152" href="./images/favicon/apple-touch-icon-152x152.png">
+  <link rel="icon" type="image/png" href="./images/favicon/favicon-196x196.png" sizes="196x196">
+  <link rel="icon" type="image/png" href="./images/favicon/favicon-96x96.png" sizes="96x96">
+  <link rel="icon" type="image/png" href="./images/favicon/favicon-32x32.png" sizes="32x32">
+  <link rel="icon" type="image/png" href="./images/favicon/favicon-16x16.png" sizes="16x16">
+  <link rel="icon" type="image/png" href="./images/favicon/favicon-128.png" sizes="128x128">
+  <meta name="application-name" content="&nbsp;">
+  <meta name="msapplication-TileColor" content="#FFFFFF">
+  <meta name="msapplication-TileImage" content="./images/favicon/mstile-144x144.png">
+  <meta name="msapplication-square70x70logo" content="./images/favicon/mstile-70x70.png">
+  <meta name="msapplication-square150x150logo" content="./images/favicon/mstile-150x150.png">
+  <meta name="msapplication-wide310x150logo" content="./images/favicon/mstile-310x150.png">
+  <meta name="msapplication-square310x310logo" content="./images/favicon/mstile-310x310.png">
+  <meta name="theme-color" content="#46483D">
+  <link rel="stylesheet" href="./css.css">
+</head>
+<body>
+  <header>
+    <div class="container">
+      <a href="./index.html">
+      <object class="header-logo" data="images/lnug-logo.svg" type="image/svg+xml"></object>
+      </a>
+
+      <nav class="header-menu">
+        <a href="./speak.html">Speak</a>
+        <a href="./sponsor.html">Sponsor</a>
+        <a href="./contact.html">Contact</a>
+      </nav>
+    </div>
+  </header>
+
+  <div id="container"><section id="gallery"><div class="single-image">
+    <a target="_blank" href="/images/gallery/11049186_1572780413006207_1571944090_n.jpg">
```

Comment:
can this just point to the origin of the image? So if pulled from flickr or instagram just link to the image on instagram?